<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeopleInfoChange</class>
 <widget class="QDialog" name="PeopleInfoChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>用户信息更改</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/myimages/icon/编辑.png</normaloff>:/myimages/icon/编辑.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>确认更改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>440</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>更改信息</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameInput">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>240</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入真实姓名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sexComboBox">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>290</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <item>
    <property name="text">
     <string>男</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>女</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="phonenumInput">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>190</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入11位手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idInput">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>340</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入18位身份证号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordInput">
   <property name="geometry">
    <rect>
     <x>298</x>
     <y>140</y>
     <width>142</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 3px;
border: 1px solid rgb(0, 0, 0);</string>
   </property>
   <property name="placeholderText">
    <string>请输入密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="sexLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>性别： </string>
   </property>
  </widget>
  <widget class="QLabel" name="phonenumLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>手机号： </string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>340</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>身份证号： </string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>密码： </string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;张海山锐谐体&quot;;</string>
   </property>
   <property name="text">
    <string>真实姓名： </string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>passwordInput</tabstop>
  <tabstop>phonenumInput</tabstop>
  <tabstop>nameInput</tabstop>
  <tabstop>sexComboBox</tabstop>
  <tabstop>idInput</tabstop>
 </tabstops>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PeopleInfoChange</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>344</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
